### DISCUSSION TITLE
POD-Galerkin ROM for N-S equations and energy equation

### DISCUSSION THREAD

**Discussion by tzuikuo** [ITHACA-FV/ITHACA-FV#479] [Q&A]
Hello!
I am a graduate student working on the POD-Galerkin reduced-order model for thermal fluid problems. I noticed that ITHACA-FV  has the tutorial of POD-Galerkin method for an unsteady Navier-Stokes problem. I wonder is there any implementation of  POD-Galerkin method for an unsteady problem accounting for both N-S equations and energy equation, so that the reduced-order model with variable U and T could be constructed.

**Comment by giovastabile** [comment]
Dear @tzuikuo,

yes, tutorials 07, 10 and 11, deal with exactly this type of problem. 07 is without buoyancy effects (i.e. weak coupling) while 10 and 11 are with buoyancy effects (strong coupling with the energy equation). It might be that you will need to restructure a bit the test case but I think it is a good starting point for your needs.

Giovanni
